lib/active_interaction/modules/validation.rb

```diff
@@ -6,12 +6,13 @@ module ActiveInteraction
   # @private
   module Validation
     class << self
-      # @param filters [Hash{Symbol => Filter}]
+      # @param interaction [Base]
       # @param inputs [Hash{Symbol => Object}]
-      def validate(filters, inputs)
+      def validate(interaction, inputs)
+        filters = interaction.class.filters
```

Comment:
This means the class has to know about how to find the filters. We could leave the `filters` arg and pass a `context`.